master/internal/project/authz_basic_impl.go

```diff
@@ -96,6 +96,17 @@ func (a *ProjectAuthZBasic) CanMoveProject(
 	return nil
 }
 
+// CanMoveProjectExperiments returns an error if the user isn't a admin or owner of a project.
+func (a *ProjectAuthZBasic) CanMoveProjectExperiments(
+	curUser model.User, from, to *projectv1.Project,
+) error {
+	if !curUser.Admin && !from.Immutable && ((curUser.ID != model.UserID(from.UserId)) ||
```

Comment:
This ```!from.Immutable``` makes it so any user can move from the default project right?

That makes sense to me just want to make sure I understand

Reply:
yes - I am asking in SaaS how they implement the default project to see whether this is the right way to reference it